### PR TITLE
Feat/1 add navigation

### PR DESCRIPTION
**Pull request by boxingnguyen**

### CODE REVIEW

File: instagram_login_screen/lib/commons/common_class.dart [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            fontSize: 14,
+            fontWeight: FontWeight.bold,
+          ),
+        ),
+      ),
+    ],
+  );
+}
+
+mixin Utils {
```
Comment (boxingnguyen) [inline instagram_login_screen/lib/commons/common_class.dart]: tách ra file khác

File: instagram_login_screen/lib/screens/Signup/new_signup.dart [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+                            return null;
+                          },
+                          onSaved: (value) {
+                            _password = value;
+                          },
+                        ),
+                      ),
+                      const SizedBox(height: 15),
+                      Padding(
+                        padding: const EdgeInsets.only(left: 5, right: 5),
```
Comment (boxingnguyen) [inline instagram_login_screen/lib/screens/Signup/new_signup.dart]: dùng symentric

File: instagram_login_screen/lib/screens/Signup/new_signup.dart [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+                            }
+                          },
+                          onSaved: (value) {
+                            _confirmPassword = value;
+                          },
+                        ),
+                      ),
+                      const SizedBox(height: 15),
+                      Padding(
+                        padding: const EdgeInsets.only(left: 5, right: 5),
```
Comment (boxingnguyen) [inline instagram_login_screen/lib/screens/Signup/new_signup.dart]: dùng symentric

File: instagram_login_screen/lib/screens/Signup/new_signup.dart [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+                        child: ElevatedButton(
+                          child: const Text(
+                            'Sign Up',
+                            style: TextStyle(
+                              color: Colors.white,
+                              fontSize: 23,
+                            ),
+                          ),
+                          onPressed: () {
+                            if (_formKey.currentState!.validate()) {
```
Comment (boxingnguyen) [inline instagram_login_screen/lib/screens/Signup/new_signup.dart]: null safety

File: instagram_login_screen/lib/screens/Signup/signup_screen.dart [hunk 5]
```diff
@@ -8,63 +9,74 @@ class SignupScreen extends StatefulWidget {
...
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: <Widget>[
+              const Icon(
+                Icons.add_circle_outlined,
+                size: 120,
+                color: Colors.white70,
+              ),
+              const SizedBox(height: 50),
+              buildTextField(text: 'Email'),
+              const SizedBox(height: 25),
```
Comment (boxingnguyen) [inline instagram_login_screen/lib/screens/Signup/signup_screen.dart]: bọc buildTextField trong Padding

File: instagram_login_screen/lib/screens/login/new_login.dart [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+                Column(
+                  children: [
+                    Image.asset(AppImg.imgLogo),
+                    const SizedBox(height: 30),
+                    Padding(
+                      padding:
+                          const EdgeInsets.only(bottom: 15, left: 5, right: 5),
+                      // child: TextFormField(
+                      //   style: const TextStyle(
+                      //       color: Colors.white60, fontSize: 20),
```
Comment (boxingnguyen) [inline instagram_login_screen/lib/screens/login/new_login.dart]: k dung thì xóa đi

File: instagram_login_screen/lib/screens/Signup/signup_screen.dart (hunk 5 above)
Comment (namapple) [reply instagram_login_screen/lib/screens/Signup/signup_screen.dart]: This 'signup_screen.dart' has been deprecated and deleted.

File: instagram_login_screen/lib/screens/login/new_login.dart (hunk 6 above)
Comment (namapple) [reply instagram_login_screen/lib/screens/login/new_login.dart]: Fixed in file!

File: instagram_login_screen/lib/screens/Signup/new_signup.dart (hunk 4 above)
Comment (namapple) [reply instagram_login_screen/lib/screens/Signup/new_signup.dart]: Fixed in file.

File: instagram_login_screen/lib/screens/Signup/new_signup.dart (hunk 3 above)
Comment (namapple) [reply instagram_login_screen/lib/screens/Signup/new_signup.dart]: Fixed in the file

File: instagram_login_screen/lib/screens/Signup/new_signup.dart (hunk 2 above)
Comment (namapple) [reply instagram_login_screen/lib/screens/Signup/new_signup.dart]: Fixed in the file

File: instagram_login_screen/lib/commons/common_class.dart (hunk 1 above)
Comment (namapple) [reply instagram_login_screen/lib/commons/common_class.dart]: Fixed in the file

**Pull request merged**